testing/libmustache/APKBUILD

```diff
@@ -0,0 +1,52 @@
+# vim: smartindent noexpandtab tabstop=4 shiftwidth=4
+# Maintainer: Renoir Boulanger <hello@renoirboulanger.com>
+
+pkgname=libmustache
+_pkghash=c809212c2941f5aa8030ce791efa38a22f43af62
+pkgver=0.4.2
+#_urlver=v${pkgver}
+_urlver=${_pkghash}
```

Comment:
Alpine style is to not use braces in variable expansions unless explicitly required -- this also applies for the instances below.